src/System.Data.Common/src/System/Xml/RegionIterator.cs

```diff
@@ -121,7 +121,7 @@ internal bool NextInitialTextLikeNodes(out string value)
             {
                 if (CurrentNode != _rowElement)
                 {
-                    Debug.Assert(false);
+                    Debug.Fail("");
```

Comment:
It looks like we have a few places where Debug.Fail doesn't show any message string. Is it worth adding something basic? Like in this case: `CurrentNode is not equal to row element`

Reply:
I've added strings to all of them.